swupd/format_switch.go

```diff
@@ -34,6 +34,9 @@ timestamp:	{{(.TimeStamp.Unix)}}
 contentsize:	{{.ContentSize -}}
 {{range .Includes}}
 includes:	{{.Name}}
+{{- end -}}
+{{range .Optional}}
+optional:	{{.Name}}
```

Comment:
I thought the format had to be changed only if the change introduced was a breaking change

Reply:
This doesn't change the format, but it does change how the manifest header will be generated based on the value of the current format. It would make sense to me that changes to the templates should only occur in the latest template or a new one should be made. @rchiossi will probably have a preference for this.

Reply:
One of the things the `format` tracks is the file format used by manifests. Right now, the correct thing is to bump the format whenever we change how the file looks like. Other than that, we never propagate those changes to previous formats. Here you would need a new template for a new format.